libs/counterfactuals/src/util/largeCounterfactualsView/calculateBubbleData.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  Cohort,
+  IDataset,
+  ifEnableLargeData,
+  IGenericChartProps,
+  JointDataset
+} from "@responsible-ai/core-ui";
+
+import { getBubbleChartOptions } from "./getBubbleChartOptions";
+
+export async function calculateBubblePlotDataFromErrorCohort(
+  errorCohort: Cohort,
+  chartProps: IGenericChartProps,
+  selectedPointsIndexes: number[],
+  customPoints: Array<{
+    [key: string]: any;
+  }>,
+  jointDataset: JointDataset,
+  dataset: IDataset,
+  isCounterfactualsDataLoading?: boolean,
+  requestBubblePlotDistribution?: (
+    request: any,
+    abortSignal: AbortSignal
+  ) => Promise<any>,
+  selectPointFromChartLargeData?: (data: any) => void,
+  onBubbleClick?: (
+    scatterPlotData: any,
+    xSeries: number[],
+    ySeries: number[],
+    indexSeries: number[]
+  ) => void,
+  onIndexSeriesUpdated?: (indexSeries?: number[]) => void
+): Promise<any | undefined> {
```

Comment:
is it possible to replace `any` with specific type , like `IHighchartsConfig`? We have a refactor work to return `any` , so trying to see if we can reduce number of any by not adding new ones

Reply:
Thanks Tong this is a great one! I removed any in as many places as possible by adding and using existing the interface.